pkg/collector/stats/stats.go

```diff
@@ -174,7 +174,7 @@ func (s *Stats) CalcDynEnergy(absM, idleM, dynM, id string) {
 
 // calcDynEnergy calculates the dynamic energy.
 func calcDynEnergy(totalE, idleE uint64) uint64 {
-	if (totalE == 0) || (idleE == 0) || (totalE < idleE) {
+	if (totalE == 0) || (totalE < idleE) {
```

Comment:
I think we should fix this bit as well .. 
If the totalE < the idle, then we have a new idle, IOW kepler over time will continue to improve its accuracy of idle by measuring the least amount of power consumed by the system.

Reply:
Agree but I think should note and fix this in the different issue and PR.